xla_native_functions.yaml

```diff
@@ -125,6 +125,7 @@ supported:
   - add.Scalar
   - add.Tensor
   - addmm
+  - affine_grid_generator
```

Comment:
Can you put this under the functionalization comment?

Reply:
Oops, good catch. Updated!